include/vptr/README.adoc

```diff
@@ -34,7 +34,7 @@ float * c = a + 5;
 assert(pMap.get_buffer(a) == pMap.get_buffer(c))
```

Comment:
I couldn't compile this line either because of the lack of operator== for buffers.

Reply:
Hmm, if the code doesn't work people might try it only to have it fail. That said, nobody has complained so far...